### ISSUE TITLE
New CSS `height` rule set as `!important` is forbidding css `resize` usage

### ISSUE THREAD

**Issue by FredZinelli** [jcubic/jquery.terminal#962]
Hello,

### Issue summary

In the last release a  css rule has been set to `!important` and it just broke the possibility to offer manual resizing to users through the `resize` css rule : since the `resize` functionality is working by assigning on the fly the height on the element itself, without using the `!important` flag, it's not possible to override the rule coming from the jQuery terminal.

### Expected behavior

When the terminal is embedded in a page, adding a `resize` css rule to it should work, like it did in previous versions.

### Actual behavior

The height of the terminal is locked and only its content (number of lines) is making the terminal's height change.

[Capture vidéo du 07-09-2024 22:48:50.webm](https://github.com/user-attachments/assets/b6f871f8-b222-449a-9dc9-6d41ddfb5706)

(you won't see my mouse, but you'll see the height value changing on the element when I try to resize it manually)

### Additional notes

The problematic rule is this one   [css/jquery.terminal.css, line 843](https://github.com/jcubic/jquery.terminal/blob/cebc63df5cdcfc22aa7df9a68a0b5b3034acb0ed/css/jquery.terminal.css#L843) (the change got published today, apparently) :

```css
@supports (--css: variables) {
    ...
    terminal {
        ...
        height: calc((var(--terminal-line) * var(--rows)) + (var(--padding, 10) * 2px)) !important;
    }
```
The comment above the rule suggests it will apply only when some variables are defined (btw, `--cols` doesn't appear in that rule), but it's not the case: I don't have a `--rows` variable defined, but it still applies and is forcing the terminal to resize depending on the number of lines of the content (without limits => none of the other calculations are applying anymore)

Can we get the `!important` removed, please?

Cheers

**Comment by jcubic** [comment]
Will check if it's possible. But is there anything that prevent you from using `!important` yourself?

**Comment by jcubic** [comment]
Thanks for the report. Will check if I can remove it. I think that I added it to overwrite inline height.

**Comment by FredZinelli** [comment]
Thx, As you can see below, this new behavior makes the UX kinda horrible in my project, so I hope you can remove it.

[Capture vidéo du 08-09-2024 00:04:29.webm](https://github.com/user-attachments/assets/884b348a-0b63-4725-849b-8817008f07c3)

**Comment by jcubic** [comment]
Will check tomorrow, for now you can use fixed `height` with `!important` and if you don't have any height use `height: auto !important`.

This also seem to break the height of the main demo on the website, so I will get rid of it.

**Comment by jcubic** [comment]
The fix was added in version 2.43.1
